torch/ao/quantization/stubs.py

```diff
@@ -21,9 +21,15 @@ def forward(self, x):
 class DeQuantStub(nn.Module):
     r"""Dequantize stub module, before calibration, this is same as identity,
     this will be swapped as `nnq.DeQuantize` in `convert`.
+
+    Args:
+        qconfig: quantization configuration for the tensor,
+            if qconfig is not provided, we will get qconfig from parent modules
     """
-    def __init__(self):
+    def __init__(self, qconfig=None):
         super(DeQuantStub, self).__init__()
+        if qconfig:
```

Comment:
should we just always attach `qconfig`? since it can be None anyways

Reply:
The current logic is that having the `qconfig` attribute not set has a meaning of "no specific qconfig for this module, use the parent", but having the `qconfig` attribute set to `None` means "do not quantize this module.  This is a bit confusing, but they are two different things.